### ISSUE TITLE
[Issue]: Blurry playback on wayland sessions

### ISSUE THREAD

**Issue by Iss-in** [jellyfin/jellyfin-media-player#340]

**Comment by iwalton3** [comment]
Looks like some type of banding. Try adding this to your `mpv.conf` ([see here](https://github.com/jellyfin/jellyfin-media-player#config-file-location)) and see if it helps:
```
deband=yes
deband-grain=0
deband-range=12
deband-threshold=32
```

**Comment by Iss-in** [comment]
added the mpv.conf in `/.local/share/jellyfinmediaplayer` with above content, no change
![image](https://user-images.githubusercontent.com/36916613/209605657-1a3c989e-717c-42db-a05c-fa1b5d31d013.png)

**Comment by paul-ri** [comment]
I'm having the same problem. Observed on 2 different machines (that have the exact same setup): Some serious colour banding when videos are played and also on the UI's background images.

More details though:

It happens only with Sway ([link](https://github.com/swaywm/sway)) `sway version 1.9-dev-2f2cdd60 (Jan 14 2023, branch 'master')`

But not on Cage ([link](https://github.com/Hjdskes/cage)) (which is how I'm mainly using Jellyfin luckily) `Cage version 0.1.4-646b3e8 (branch 'makepkg')`

> This is Cage, a Wayland kiosk. A kiosk runs a single, maximized application.

It took me a while to finally find that the issue was related to Wayland and then this issue!

To summarize:
- X11 session: no banding
- Cage (uses Wayland): No banding
- Sway
  - Jellyfin: banding
  - mpv: No banding
  - VLC: No banding

tweaking the `mpv.conf` file just creates sub-par results. I tried adding and tweakings these banding thresholds a bit, it's just always bad and I figured this wasn't the right approach anyway.

**Comment by paul-ri** [comment]
Well, that was quick. The following fixes the issue:

`QT_WAYLAND_DISABLE_WINDOWDECORATION=0 jellyfinmediaplayer`

First result from an online search for `wayland colour banding` yields https://bugreports.qt.io/browse/QTBUG-99473
